pages/implementation/destinations/segment.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: 'guide'
+title: 'Avo Functions to Segment'
+---
+
+import Link from '../../../components/Link';
+
+- <Link passHref href="/implementation/destinations/segment-javascript"><a>JavaScript</a></Link>
+- <Link passHref href="/implementation/destinations/segment-kotlin"><a>Kotlin</a></Link>
+- <Link passHref href="/implementation/destinations/segment-swift"><a>Swift</a></Link>
```

Comment:
This list indicates that Segment is not available in other languages, which is something we should avoid. While we don't have destination specific docs for each language, should we link to a more generic custom destination docs for all the other languages?